### DISCUSSION TITLE
Does gauge runs on Apache?

### DISCUSSION THREAD

**Discussion by manavghaigit** [getgauge/gauge#1838] [General]
Does gauge runs on apache or uses apache in any way? We would like to bring gauge into our company but while it through security team they mentioned it runs on apache. Can you please shed some light on this? And would be helpful if there is any documentation around it.

**Comment by sriv** [comment]
If you mean the [Apache HTTP Server](https://httpd.apache.org/) which is often referred to as the Apache Web Server or just Apache, then no, Gauge does not use it.

Gauge and it's plugins are shipped under Apache 2.0 License
